patches/0001-Fix-gnmi_cli-hang.patch

```diff
@@ -0,0 +1,47 @@
+From be4d7a0f0c902fc5fc7cc4dbc2f8342679b4daf9 Mon Sep 17 00:00:00 2001
+From: zbud-msft <zainbudhwani@microsoft.com>
+Date: Wed, 4 Jan 2023 00:07:04 +0000
+Subject: [PATCH] Fix gnmi_cli hang
+
```

Comment:
Please add comment to explain your patch

Reply:
Fixed last patch by removing writing to channel inside cli_config. Combined fixes into existing patch